### PR TITLE
ADD: Tap, and hold, on balance to display a Copy balance option

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -85,6 +85,7 @@
     "react-native-sortable-list": "0.0.22",
     "react-native-svg": "9.1.1",
     "react-native-tcp": "3.3.0",
+    "react-native-tooltip": "^5.2.0",
```
Comment (Overtorment) [inline package.json]: not pinned

File: screen/wallets/transactions.js [hunk 2]
```diff
@@ -222,7 +229,17 @@ export default class WalletTransactions extends Component {
         >
           {this.state.wallet.getLabel()}
         </Text>
-        <TouchableOpacity onPress={() => this.changeWalletBalanceUnit()}>
+        {Platform.OS === 'ios' && (
```
Comment (Overtorment) [inline screen/wallets/transactions.js]: doesn't work on android..?

Comment (marcosrdz) [reply screen/wallets/transactions.js]: Just added

File: package.json (hunk 1 above)
Comment (marcosrdz) [reply package.json]: Fixed

**Pull request merged**